conf/st2.conf.sample

```diff
@@ -100,6 +100,16 @@ logging = conf/logging.exporter.conf
 # Directory to dump data to.
 dump_dir = /opt/stackstorm/exports/
 
+[garbagecollector]
```

Comment:
If you'd like to have it as a separate service, it could be nice to bundle it into st2common package...

Plus if you want to enabled/disabled flag it should be better controll it on os level, rather then config. `/etc/default/st2garbage..` or `/etc/sysconfig/st2garbage..`, imo.